.prettierrc

```diff
@@ -1,9 +1,9 @@
 {
-  "trailingComma": "es5",
-  "tabWidth": 2,
-  "semi": true,
-  "singleQuote": false,
-  "htmlWhitespaceSensitivity": "strict",
-  "plugins": [],
-  "printWidth": 140
-}
\ No newline at end of file
+	"trailingComma": "es5",
+	"tabWidth": 2,
```

Comment:
It seems Github displays tabs with 8 spaces which looks too much, I think it's better if we stick with spaces instead of tabs

Reply:
> It seems Github displays tabs with 8 spaces which looks too much, I think it's better if we stick with spaces instead of tabs

Your request seems fine (I just updated the config.)

I wonder if there is a better way to keep all of our template settings in sync across all of our repos, because it has been tedious for me to update all of these configs across all of the active repositories whenever changes like this are made.